front/src/modules/timesStops/helpers/utils.ts

```diff
@@ -114,7 +114,7 @@ const getDigits = (unit: string | undefined) =>
   unit === MarginUnit.second || unit === MarginUnit.percent ? 0 : 1;
 
 export function formatDigitsAndUnit(fullValue: string | number | undefined, unit?: string) {
-  if (fullValue === undefined || fullValue === '0%') {
+  if (fullValue === undefined) {
```

Comment:
Why did you remove the "0%" here ?

Reply:
Because according to the last spec, I believe we want to display all margin values, even if they are equal to 0%